crates/egui-winit/src/lib.rs

```diff
@@ -426,6 +426,11 @@ impl State {
                 self.egui_input.modifiers.ctrl = ctrl;
                 self.egui_input.modifiers.shift = shift;
                 self.egui_input.modifiers.mac_cmd = cfg!(target_os = "macos") && super_;
+                self.egui_input.modifiers.meta = if cfg!(target_os = "macos") {
+                    false
+                } else {
+                    super_
+                };
```

Comment:
I think this is the path of least surprise, considering this is also how the meta key is defined on web.

```suggestion
                self.egui_input.modifiers.meta = super_;
```